devops/scripts/benchmarks/utils/unitrace.py

```diff
@@ -0,0 +1,182 @@
+                print(f"Removing old unitrace dir: {full_path}")
+                shutil.rmtree(full_path)
+
+
+def unitrace_cleanup(bench_cwd, unitrace_output):
+    # Remove .pid files from the benchmark directory and .json files from cwd
+    unitrace_dir = os.path.dirname(unitrace_output)
+    unitrace_base = os.path.basename(unitrace_output)
+    print(f"Cleanup unitrace output {unitrace_base} from {unitrace_dir}")
+    for f in os.listdir(unitrace_dir):
+        if f.startswith(unitrace_base + "."):
+            os.remove(os.path.join(unitrace_dir, f))
+            print(f"Cleanup: Removed {f} from {unitrace_dir}")
+    if os.path.exists(bench_cwd):
+        for f in os.listdir(bench_cwd):
+            if f.endswith(".json"):
+                os.remove(os.path.join(bench_cwd, f))
+                print(f"Cleanup: Removed {f} from {bench_cwd}")
+
+
+def unitrace_prepare(name, unitrace_timestamp, command, extra_unitrace_opt=[]):
+    unitrace_bin = os.path.join(options.workdir, "unitrace-build", "unitrace")
+    if not os.path.exists(unitrace_bin):
+        raise FileNotFoundError(f"Unitrace binary not found: {unitrace_bin}. ")
+    os.makedirs(options.unitrace_res_dir, exist_ok=True)
+    if not options.save_name:
+        raise ValueError(
+            "Unitrace requires a save name to be specified via --save option."
+        )
+    bench_dir = f"{options.unitrace_res_dir}/{options.save_name}_{unitrace_timestamp}"
+    os.makedirs(bench_dir, exist_ok=True)
+
+    unitrace_output = f"{bench_dir}/{name}_{unitrace_timestamp}"
+    unitrace_command = (
+        [
+            str(unitrace_bin),
+            "--call-logging",
+            "--host-timing",
+            "--device-timing",
+            "--chrome-sycl-logging",
+            "--chrome-call-logging",
+            "--chrome-kernel-logging",
+            "--output",
+            unitrace_output,
+        ]
+        + extra_unitrace_opt
+        + command
+    )
+    if options.verbose:
+        print(f"Unitrace cmd: {' '.join(unitrace_command)}")
+
+    return bench_dir, unitrace_output, unitrace_command
+
+
+def handle_unitrace_output(bench_dir, unitrace_output, timestamp):
+    # Handle unitrace_output.{pid} logs: rename to unitrace_output (remove pid)
+    for f in os.listdir(bench_dir):
+        if f.startswith(os.path.basename(unitrace_output) + "."):
+            parts = f.rsplit(".", 1)
+            if (
+                len(parts) == 2
+                and parts[1].isdigit()
+                and os.path.isfile(os.path.join(bench_dir, f))
+            ):
+                src = os.path.join(bench_dir, f)
+                dst = os.path.join(bench_dir, os.path.basename(unitrace_output))
+                shutil.move(src, dst)
+                if options.verbose:
+                    print(f"Renamed {src} to {dst}")
+                break
+
+    # Handle {name}.{pid}.json files in cwd: move and rename to {self.name()}_{timestamp}.json
```

Comment:
Doesn't `unitrace` already have an option for changing the default name of jsons and `.pid` logs?